gamemodes/terrortown/gamemode/client/cl_help/populate_bindings.lua

```diff
@@ -0,0 +1,89 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/bindings")
+
+local function AddBindingCategory(category, parent)
+	local client = LocalPlayer()
+
+	local form = CreateForm(parent, category)
+
+	local bindings = bind.GetSettingsBindings()
+
+	for i = 1, #bindings do
+		local binding = bindings[i]
+
+		if binding.category ~= category then continue end
+
+		local currentBinding = bind.Find(binding.name)
+
+		form:MakeBinder({
+			label = binding.label,
+			select = currentBinding,
+			default = binding.defaultKey,
+			onDisable = function(slf, binder)
+				bind.Remove(currentBinding, binding.name, true)
+
+				binder:SetValue(bind.Find(binding.name))
+			end,
+			onChange = function(slf, keyNum)
+				if currentBinding == keyNum then return end
+
+				bind.Remove(currentBinding, binding.name, true)
+
+				if keyNum ~= 0 then
+					bind.Add(keyNum, binding.name, true)
+				end
+
+				local key = keyNum ~= 0 and string.upper(input.GetKeyName(keyNum)) or LANG.GetTranslation("button_none")
+
+				client:ChatPrint(
+					LANG.GetParamTranslation("ttt2_bindings_new", {
+						name = binding.name,
+						key = key
+					})
+				)
+
+				currentBinding = keyNum
```

Comment:
Neccessary?

Reply:
sam here. It has been this way all the time, I just transformed it into the new system